docs/CHANGELOG.md

```
@@ -2,7 +2,54 @@
 
 _What's new across RunMat. See [GitHub Releases](https://github.com/runmat-org/runmat/releases) for runtime release binaries._
 
-## [v0.4.9](https://github.com/runmat-org/runmat/compare/v0.4.8...v0.4.9)
+## [v0.5.0 ](https://github.com/runmat-org/runmat/compare/v0.4.9...v0.5.0) - June 2026
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)